bigquery_etl/pytest_plugin/udf.py

```diff
@@ -52,7 +50,7 @@ class UdfFile(pytest.File):
     def collect(self):
         """Collect."""
         self.add_marker("udf")
-        self.udf = parsed_udfs()[self.name]
+        self.udf = parsed_routines()[self.name]
```

Comment:
I think that this general change in `s/udf/routine` should update this file as well, since we are relying on it for procedure testing.